telegram/_replykeyboardmarkup.py

```diff
@@ -41,12 +41,12 @@ class ReplyKeyboardMarkup(TelegramObject):
             each represented by an Array of :class:`telegram.KeyboardButton` objects.
         resize_keyboard (:obj:`bool`, optional): Requests clients to resize the keyboard vertically
             for optimal fit (e.g., make the keyboard smaller if there are just two rows of
-            buttons). Defaults to :obj:`False`, in which case the custom keyboard is always of the
+            buttons). Defaults to :obj:`None`, in which case the custom keyboard is always of the
```

Comment:
I think I would prefer to keep this as is, i.e. document what TG does when the parameter is not passed. (BTW otherwise, we'd have to change also all the other docstrings)